modules/metastorage-api/src/main/java/org/apache/ignite/internal/metastorage/Entry.java

```diff
@@ -18,58 +18,44 @@
 package org.apache.ignite.internal.metastorage;
 
 import java.io.Serializable;
+import org.apache.ignite.internal.hlc.HybridTimestamp;
 import org.jetbrains.annotations.Nullable;
 
-/**
- * Represents a storage unit as entry with key, value and revision.
- *
- * <p>Where:
- * <ul>
- *     <li>key - an unique entry's key. Keys are comparable in lexicographic manner.</li>
```

Comment:
Why is this `<ul>` removed? It explains nicely what type of entries there are. Including that ill-named `empty`. It seems useful.

Reply:
It does not reflect the entire contents of the interface, in general you can read the fields and understand what is in it.
Do you think that it should be returned?

Reply:
Yes, please return it. It helps to have a summary on the interface, it's more difficult to gather information scattered by documentation of fields and methods.

Reply:
fix it